<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet Fin d'etude</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/settings_1.png</normaloff>icons/settings_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/home-189.png</normaloff>icons/home-189.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>721</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>581</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Encodage</string>
       </attribute>
       <widget class="QLineEdit" name="in_encodage">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>331</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Waree</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter un nom pour le fichier d'encodage :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_encodage">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>110</y>
          <width>51</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/me.png</normaloff>icons/me.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>501</width>
          <height>291</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>499</width>
           <height>289</height>
          </rect>
         </property>
         <widget class="QTextEdit" name="label_3_">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>501</width>
            <height>291</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="btnSelect">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>71</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/home_download.png</normaloff>icons/home_download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sélectionner DataSet :</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Reconnaissance Faciale</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>301</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sélectionner le fichier d'encodage :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_pickle">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>40</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/home_download.png</normaloff>icons/home_download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>500</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tlwg Mono</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Démarrer l'application</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_reco">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>370</y>
          <width>151</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/face_reco.png</normaloff>icons/face_reco.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Classificateur d'images</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sélectionner le fichier d'encodage </string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSelect_1">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/home_download.png</normaloff>icons/home_download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_path_claf">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>110</y>
          <width>61</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/folder.png</normaloff>icons/folder.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Chenmin à votre images </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>pour classifier     </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>260</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Chenmin à un  dossier pour</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>300</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string> sauvegarder les résultats  </string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_path_output">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>270</y>
          <width>71</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/folder.png</normaloff>icons/folder.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="classification_btn">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>410</y>
          <width>91</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/classification.png</normaloff>icons/classification.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>490</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tlwg Mono</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Classificateur d'images</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Contrôleur à l'entrée</string>
       </attribute>
       <widget class="QPushButton" name="btnSelect_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/home_download.png</normaloff>icons/home_download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sélectionner le fichier d'encodage </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vérifier l'existence </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>250</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tibetan Machine Uni</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>les fichiers de détection </string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_detct">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>210</y>
          <width>71</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/check.png</normaloff>icons/check.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_track">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>350</y>
          <width>151</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/tracking.png</normaloff>icons/tracking.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>480</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tlwg Mono</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Démarrer l'application</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>551</height>
       </rect>
      </property>
      <widget class="QPushButton" name="btnshow">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/vieuDB.png</normaloff>icons/vieuDB.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="delet_btn">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>230</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/deletDb.png</normaloff>icons/deletDb.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>230</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tibetan Machine Uni</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Voir les données </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>260</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tibetan Machine Uni</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>de Contrôleur</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>240</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tibetan Machine Uni</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Supprimer données </string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/information.png</normaloff>icons/information.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tibetan Machine Uni</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A propos de l'application</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/settings-flat.png</normaloff>icons/settings-flat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
